src/main/java/Problem1.java

```diff
@@ -1,10 +1,39 @@
-public class Problem1 {
+/**
+ * The code orders the the array so that all the 0s are on the left, the 1s are in the middle, and the 2s are at
```

Comment:
no need to describe what the requirement is for this section. only use the sentences for your design